docs/deployment/on_kubernetes.md

```diff
@@ -5,3 +5,97 @@
 </div>
 
 # Running Kyuubi on Kubernetes
+
+## Requirements
+
+When you want to deploy Kyuubi’s Spark SQL engines on Kubernetes(use K8S as the name below), you’d better have cognition
```

Comment:
Deploy engine or deploy Kyuubi ? I guess we should support both, and we need document these.

But if this doc is about deply engine, it's better to specify at title and description.

Reply:
Maybe, i can add some about deploy kyuubi in this.